src/Network/Receive.pm

```diff
@@ -435,6 +645,427 @@ sub connection_refused {
+			$messageSender->sendSync(1);
+			$initSync = 0;
+		}
+	},
+	VAR_MAXJOBEXP, sub {
+		$_[0]{exp_job_max_last} = $_[0]{exp_job_max};
+		$_[0]{exp_job_max_last2} = $_[0]{exp_job_max} if !$_[0]{exp_job_max_last2};
+		$_[0]{exp_job_max} = $_[1];
+		#message TF("BaseExp: %s | JobExp: %s\n", $monsterBaseExp, $monsterJobExp), "info", 2 if ($monsterBaseExp);
+	},
+	VAR_WEIGHT, sub { $_[0]{weight} = $_[1] / 10 },
+	VAR_MAXWEIGHT, sub { $_[0]{weight_max} = int($_[1] / 10) },
+	#VAR_POISON
+	#VAR_STONE
+	#VAR_CURSE
+	#VAR_FREEZING
+	#VAR_SILENCE
+	#VAR_CONFUSION
+	VAR_STANDARD_STR, sub { $_[0]{points_str} = $_[1] },
+	VAR_STANDARD_AGI, sub { $_[0]{points_agi} = $_[1] },
+	VAR_STANDARD_VIT, sub { $_[0]{points_vit} = $_[1] },
+	VAR_STANDARD_INT, sub { $_[0]{points_int} = $_[1] },
+	VAR_STANDARD_DEX, sub { $_[0]{points_dex} = $_[1] },
+	VAR_STANDARD_LUK, sub { $_[0]{points_luk} = $_[1] },
+	#VAR_ATTACKMT
+	#VAR_ATTACKEDMT
+	#VAR_NV_BASIC
+	VAR_ATTPOWER, sub { $_[0]{attack} = $_[1] },
+	VAR_REFININGPOWER, sub { $_[0]{attack_bonus} = $_[1] },
+	VAR_MAX_MATTPOWER, sub { $_[0]{attack_magic_max} = $_[1] },
+	VAR_MIN_MATTPOWER, sub { $_[0]{attack_magic_min} = $_[1] },
+	VAR_ITEMDEFPOWER, sub { $_[0]{def} = $_[1] },
+	VAR_PLUSDEFPOWER, sub { $_[0]{def_bonus} = $_[1] },
+	VAR_MDEFPOWER, sub { $_[0]{def_magic} = $_[1] },
+	VAR_PLUSMDEFPOWER, sub { $_[0]{def_magic_bonus} = $_[1] },
+	VAR_HITSUCCESSVALUE, sub { $_[0]{hit} = $_[1] },
+	VAR_AVOIDSUCCESSVALUE, sub { $_[0]{flee} = $_[1] },
+	VAR_PLUSAVOIDSUCCESSVALUE, sub { $_[0]{flee_bonus} = $_[1] },
+	VAR_CRITICALSUCCESSVALUE, sub { $_[0]{critical} = $_[1] },
+	VAR_ASPD, sub {
+		$_[0]{attack_delay} = $_[1] >= 10 ? $_[1] : 10; # at least for mercenary
+		$_[0]{attack_speed} = 200 - $_[0]{attack_delay} / 10;
+	},
+	#VAR_PLUSASPD
+	VAR_JOBLEVEL, sub {
+		my ($actor, $value) = @_;
+
+		$actor->{lv_job} = $value;
+		message sprintf($actor->verb("%s are now job level %d\n", "%s is now job level %d\n"), $actor, $actor->{lv_job}), "success", $actor->isa('Actor::You') ? 1 : 2;
+
+		return unless $actor->isa('Actor::You');
+		
+		Plugins::callHook('job_level_changed', {
+			level	=> $actor->{lv_job}
+		});
+
+		if ($config{dcOnJobLevel} && $actor->{lv_job} >= $config{dcOnJobLevel}) {
+			message TF("Disconnecting on job level %d!\n", $config{dcOnJobLevel});
+			chatLog("k", TF("Disconnecting on job level %d!\n", $config{dcOnJobLevel}));
+			quit();
+		}
+	},
+	#...
+	VAR_MER_KILLCOUNT, sub { $_[0]{kills} = $_[1] },
+	VAR_MER_FAITH, sub { $_[0]{faith} = $_[1] },
+	#...
+);
+
+sub stat_info {
+	my ($self, $args) = @_;
+
+	return unless changeToInGameState;
+
+	my $actor = {
+		'00B0' => $char,
+		'00B1' => $char,
+		'00BE' => $char,
+		'0141' => $char,
+		'01AB' => exists $args->{ID} && Actor::get($args->{ID}),
+		'02A2' => $char->{mercenary},
+		'07DB' => $char->{homunculus},
+		'0ACB' => $char,
+		'081E' => exists $args->{ID} && Actor::get($args->{ID}), # Sorcerer's Spirit
```

Comment:
Maybe $char->{Elemental}, ?

Reply:
can be other elemental fom another socerer